### PR TITLE
fix: add new i18n keys, data management page and Update de.json

### PR DESCRIPTION
**Pull request by ReuschelCGN**

### CODE REVIEW

**Comment by TurtIeSocks** [comment]
Oops, good catches. Thanks! Awaiting your review, @Fabio1988

File: packages/locales/lib/human/de.json [hunk 1]
```diff
@@ -704,5 +704,35 @@
   "gender_filters_all": "Geschlecht gilt für alle",
   "and_caption": "miteinander verbunden",
   "or_caption": "filtert unabhängig",
-  "filter_help": "Filter Hilfe"
+  "filter_help": "Filter Hilfe",
+  "kilometers": "Kilometer",
+  "miles": "Miles",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:709]: Meilen

File: packages/locales/lib/human/de.json [hunk 2]
```diff
@@ -704,5 +704,35 @@
...
+  "distance_unit": "Entfernungseinheit",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:710]: Maßeinheit

File: packages/locales/lib/human/de.json [hunk 3]
```diff
@@ -704,5 +704,35 @@
...
+  "data_management": "Einstellungsverwaltung",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:711]: Datenverwaltung

Or do you think it's more intuitive to name it like you said?

File: packages/locales/lib/human/de.json [hunk 4]
```diff
@@ -704,5 +704,35 @@
...
+  "reset_icons": "Icons zurücksetzen",
+  "reset_position": "Position zurücksetzen",
+  "reset_ui": "UI zurücksetzen",
+  "reset_options": "Optionen zurücksetzen",
+  "reset_menus": "Menüs zurücksetzen",
+  "reset_settings": "Einstellungen zurücksetzen",
+  "reset_general": "Allgemein",
+  "reset_assets": "Styles",
+  "reset_filters": "Filter",
+  "reset_pokemon": "Pokémons zurücksetzen",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:724]: There is no "Pokémons", only plural "Pokémon"

**Comment by ReuschelCGN** [comment]
think the translation is fine now @TurtIeSocks

**Review by Fabio1988 — APPROVED** [review]

**Pull request merged**